mod/cli/pkg/utils/deposit/BeaconDepositContract.go

```diff
@@ -0,0 +1,1021 @@
+// Code generated - DO NOT EDIT.
```

Comment:
Noticed don't have to generate the go bindings for the Deposit contract, it is already there. I could re-use it.